test/e2e/cassandracluster_test.go

```diff
@@ -405,8 +405,8 @@ func cassandraClusterCleanupTest(t *testing.T, f *framework.Framework, ctx *fram
 			return false, nil
 		}
 
-		if !endTime.After(startTime) {
-			t.Logf("Expected endTime (%s) to be after startTime (%s)", endTime, startTime)
+		if endTime.Sub(startTime) < 0 {
```

Comment:
Why did you change this test ?

Reply:
because some time it runs even in the same second so startTime = endTime and the test fails.